calculator.rb

```diff
@@ -0,0 +1,122 @@
+# welcome message
+puts "welcome to the Calculator program! which operator would you like to use?
```

Comment:
Just as a style suggestion, put all the methods at the top and all the main driver lines at the bottom.  That can make the code easier to read and follow.